cog-book/source/references/greenbar_tags.rst

```diff
@@ -231,7 +232,7 @@ Any other attributes will be interpreted as custom fields and included
 in the attachments' ``fields`` field. Custom fields have the following
 structure:
 
-::
+.. code-block: json
```

Comment:
Think this should be two colons instead of one?

Reply:
Nice catch.